hedyorg/hedy
# 🪲 Fix make_response incorrectly sending `text` instead of JSON

#5558 Introduced an error where some calls where expecting a JSON, and got text instead. I checked the places where this happened and fix the calls. 

If I'm missing some, please add them 😄 

**How to test**
Perform the following: 

* Update an adventure
* Update  a class customization
* Create students account
* Reset password
* Reset a student's password
* Update your public profile

And all of them should show the success modal! 

Also:
Fixes #5550
